src/Symfony/Component/VarDumper/Dumper/CliDumper.php

```diff
@@ -613,4 +615,16 @@ private function getSourceLink(string $file, int $line)
 
         return false;
     }
+
+    /**
+     * Replace invisible character by the hexadecimal representation of the codepoint.
+     */
+    private function replaceInvisibleCharacters(string $input): string
+    {
+        $invisibleCharacters = ['﻿'];
```

Comment:
![image](https://user-images.githubusercontent.com/9253091/209000576-7317f135-84f9-4474-8f93-0a1d5b234e37.png)

- I'm not sure about which chars we must replace https://invisible-characters.com/
- List of codepoint should be better, but I don't know how we can `preg_replace` with these

Reply:
Can't we use `CliDumper::$controlCharsMap` instead of adding new code?
The invisible characters should be written as `"\u{FEFF}"`.

Reply:
`$controlCharsRx = '/[\x00-\x1F\x7F]+/';` should be also transformed to `$controlCharsRx = '/[\x00-\x1F\x7F\x{feff}]+/u'`

But I don't know/understand how adapt this following part
https://github.com/symfony/symfony/blob/e9870a418619b296cafe67c4f8d1d0eb2dbd0ce5/src/Symfony/Component/VarDumper/Dumper/CliDumper.php#L443-L451

Without editing output will be `bo\xEF\xBB\xBFm` instead of `bo\u{FEFF}m`. It's try access 3 times in `controlCharsMap` not only one